main.css

```diff
@@ -0,0 +1,235 @@
+body{
+    
+    width:1440px;
+    height: 100%;
+    margin: auto;
+    
+}
+#main {
+  border: .01px transparent solid;
+  width: 1440px;
+  height:  805px;
```

Comment:
This line is affecting your code, the footer jumps up. You may want to look into heights and width, most times you don't need to explicitly set heights of a large container. and your width is off too, it leaves a big space that is scrollable, too big to be ignored.